right/src/slave_drivers/uhk_module_driver.c

```diff
@@ -24,6 +24,11 @@ uint8_t UhkModuleSlaveDriver_DriverIdToSlotId(uint8_t uhkModuleDriverId)
     return uhkModuleDriverId+1;
 }
 
+void UhkModuleSlaveDriver_ResetTrackpoint()
+{
+    UhkModuleStates[UhkModuleDriverId_RightModule].phase = UhkModulePhase_ResetTrackpoint;
```

Comment:
I guess writing phase directly could mess up module initialization (if resetTrackpoint is called during initialization phase). Will add an indirection in order to fix it...